package.xml

```diff
@@ -4,7 +4,7 @@
   <name>tinyxml_vendor</name>
   <version>0.8.2</version>
   <description>CMake shim over the tinxml library.</description>
-  <maintainer email="michel@ekumenlabs.com">Michel Hidalgo</maintainer>
```

Comment:
As a courtesy, we usually take the old maintainers and move them to an "author" tag to recognize their work in maintenance.  I'll suggest that here.

Reply:
Done in ba8b977.